src/components/Language.js

```diff
@@ -19,9 +19,10 @@ export const Language = () => {
       const itemIndex = index - 1
       const item = items[itemIndex]
 
-      i18n.setLocale(item.lang)
+      // i18n.setLocale(item.lang)
       setLang(i18n.locale)
-      setAppLanguage(item.lang)
+      // setAppLanguage(item.lang)
```

Comment:
[Waiting for confirmation](https://phabricator.wikimedia.org/T248080#5988590) that we'd be removing the ability for users to change app language altogether, that would mean some lines in `Language.js` get deleted